build/ci/e2e/GKE_k8s_versions.jenkinsfile

```diff
@@ -96,5 +97,6 @@ overrides:
   gke:
     gCloudProject: $GCLOUD_PROJECT
 EOF
+        make -C build/ci COMMAND=ci-e2e ci
```

Comment:
Could we store both `environment` and `run-config.yml` files in `ci/e2e/` (close to the Jenkinsfile), pre-populated?

Each Jenkins job would come up with 3 files:
- Jenkinsfile
- run-config.yml  (I'd prefer deployer-config.yml)
- environment (I'd prefer .env)

Then, in the Jenkinsfile, call something like `envsubst` here to set environment variables automatically from the file content (and maybe output that into another file in the root dir).

I think this would look a bit cleaner, and would also simplify the "manual" process of copying this content into a file when running CI stuff locally.

Reply:
We chatted offline and concluded to not introduce this change right now. 

While it would improve our ability to repro and made the job file cleaner, it would also mean we are back to having settings spread to multiple files. We would also need to coordinate variable names between those files (to pass gcloud_project, vault_addr, etc.). To include these files we would need to use some kind of jenkins plugin or keep self links in the job file, which is probably more involved than what we currently have. 

As the settings file grow or are parametrized in more dimensions, we might want to revisit this.